stdpopsim/qc/HomSap.py

```diff
@@ -1565,3 +1565,27 @@ def extended_pulse(
 _species.get_demographic_model(
     "OutOfAfricaExtendedNeandertalAdmixturePulse_3I21"
 ).register_qc(Iasi2021())
+
+def Huber2017():
+    """
+    Gamma DFE from Huber et al. 2017 PNAS.
+    """
+   
+    id = "Huber2017_gamma_dfe"
+    neutral = stdpopsim.MutationType()
+    negative = stdpopsim.MutationType(
+        dominance_coeff=0.5,
+        distribution_type="g",  # gamma distribution
+        distribution_args=[-0.014, 0.19],
+    )
+    
+    return stdpopsim.DFE(
+        id=id,
+        description=id,
+        long_description=id,
+        mutation_types=[neutral, negative],
+        proportions=[0.3, 0.7],
+    )
+
+
+_species.get_dfe("Gamma_H17").register_qc(Huber2017())
```

Comment:
Hi @petrelharp 

Should I remove these codes and open another pr for QC this DFE and other DFEs?

Reply:
I think it's important to leave this QC in here, because we should be able to see that your QC framework works.